tests/tests_xml.py

```diff
@@ -302,7 +302,7 @@ def test_xml_enum(self):
         self._test_xml_var_type(o, "enum")
 
     def test_xml_enumvalues(self):
-        o = self.opc.nodes.objects.add_variable(2, "xmlenumvalues", 0, varianttype=ua.VariantType.Int32, datatype=ua.ObjectIds.AttributeWriteMask)
+        o = self.opc.nodes.objects.add_variable(2, "xmlenumvalues", 0, varianttype=ua.VariantType.UInt32, datatype=ua.ObjectIds.AttributeWriteMask)
```

Comment:
In fact that test is broken. AttributeWriteMask does not seem to inherit writemaskanymore...